rotkehlchen/tests/api/test_liquity.py

```diff
@@ -275,7 +277,10 @@ def test_staking_stats(rotkehlchen_api_server, ethereum_accounts):
     assert len(global_stats['staking_gains']) == 3
     assert len(global_stats['stability_pool_gains']) == 2
     assert FVal(global_stats['stability_pool_gains'][0]['amount']).is_close(FVal(address_0_data['stability_pool_gains'][0]['amount']) + FVal(address_1_data['stability_pool_gains'][0]['amount']), max_diff='1e-8')  # noqa: E501
-    assert address_0_data['stability_pool_gains'][0]['amount'] == '1.7820064710306824'
+    assert math.isclose(
+        float(address_0_data['stability_pool_gains'][0]['amount']), 1.7820064710306824,
+        rel_tol=1e-15,
+    )
```

Comment:
Windows yields `1.7820064710306822`. I'd put that down to floating point precision.